shared/src/tracing.rs

```diff
@@ -2,19 +2,30 @@ use std::{
     panic::{self, PanicInfo},
     thread,
 };
-use tracing_subscriber::fmt::time::ChronoUtc;
+use tracing::level_filters::LevelFilter;
+use tracing_subscriber::fmt::{time::ChronoUtc, writer::MakeWriterExt as _};
 
 /// Initializes tracing setup that is shared between the binaries.
 /// `env_filter` has similar syntax to env_logger. It is documented at
 /// https://docs.rs/tracing-subscriber/0.2.15/tracing_subscriber/filter/struct.EnvFilter.html
-pub fn initialize(env_filter: &str) {
+pub fn initialize(env_filter: &str, stderr_threshold: LevelFilter) {
```

Comment:
I would slightly prefer: 
```suggestion
pub fn initialize(env_filter: &str, stderr_threshold: Option<Level>) {
```